boskos/common/common.go

```diff
@@ -19,9 +19,10 @@ package common
 import (
 	"errors"
 	"fmt"
-	"gopkg.in/yaml.v2"
 	"strings"
 	"time"
+
+	"gopkg.in/yaml.v2"
```

Comment:
We've been using ghodss/yaml most places

Reply:
I have another PR to fix this. This is just fixing the format